check-versions.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2021 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+# exit when any command fails
+set -e
+
+# log aio version
+aio info
+
+# log imagemagick version
+echo "....................................................................................................."
+identify --version
+echo "....................................................................................................."
+
+
+# log exiftool version
+echo "....................................................................................................."
+echo "Exiftool Version:"
+exiftool -ver
+echo "....................................................................................................."
+
+met=$(exiftool README.md -json | grep "FileSize")
+if [[ -z "$met" ]]; then
+    echo "WARNING: Exiftool not installed."
+fi
```

Comment:
Can remove this? `met=$(exiftool README.md -json | grep "FileSize")`

Reply:
Yep, leftover from before I discovered how to log exiftool version. Nice catch!